techmatters/terraso-mobile-client
# Feat: Soil ID sheet exists for site or temporary location

## Description
- Create the Soil ID Sheet and implement most of its static content.
- Also includes some minor refactoring, like renaming SiteScreen to LocationDashboardContent and extracting CreateSiteButton as a component

### Checklist
- [ ] New tests added -- Did not do this, don't think snapshot tests are valuable at this point

### Related Issues
#1196

### Verification steps
1. Press the "Explore the data >" button from a site 
Confirm the components that don't require Soil ID or site data are there
2. Press the "Explore the data >" button from a temporary location
Confirm the components that don't require Soil ID or site data are there
